career/elevator_pitch.md

```diff
@@ -0,0 +1,68 @@
+Whenever you are in a position where you quickly need to tell someone about yourself with limited time, then the elevator pitch is an excellent tool.
+
+The following paragraph (shamelessly taken from wikipedia) sums it up quite nicely;
+
+> An elevator pitch, elevator speech, or elevator statement is a short
+> description of an idea, product, company, or oneself that explains the
+> concept in a way such that any listener can understand it in a short
+> period of time.
+
+This description typically explains who the person is, what he/she does, and if he or she has the relevant experience. For the latter, when explaining an individual, the description generally explains one’s skills and goals, and why they would be a productive and beneficial person to have on a team or within a company/project.
+
+An elevator pitch does not have to include all of these components. But it usually has at least some explanations of what the idea, product, company, or person is and why it/they are valuable.
+
+
+Your assignment is to create a elevator pitch for when you are networking and meeting recruiters. The pitch should sound natural and have an easy flow. The pitch should not sound mechanical or rehearsed, even though you will rehearse it over and over again.
+
+  
+  
+
+The following steps are useful when creating a good elevator pitch
+
+ 
+
+## Target your pitch
+
+
+
+  
+
+You need to think about who your target audience is. There is a difference between general networking and talking directly to recruiters.
+
+Depending on who you’re speaking with you need to adapt the pitch for that specific person/target. You can create multiple elevator pitches in order to have them ready for different targets.
+
+## Answer important questions
+
+There is some information that you need to emphasize in order to grab for example a recruiters attention.
+
+A good pitch should answer three main questions that the target could have:
+
+-   Who are you?
+    
+-   What are you looking for?
+    
+-   What does the listener gain in hearing you out?
+    
+
+These questions formulated in different ways are important for us to bring forward. The first question gives the target general information about you and a general idea if you would be a fit for the company. The second one answers the question on why you are talking to that specific target. And the third question answers how the target could benefit from getting to know you better or potentially hiring you. These questions should be answered in an natural way with a natural flow. Here is an example of how a pitch could go:
```

Comment:
`a general idea if you would be a fit for the company.` good fit 
`from getting to know you better or potentially hiring you.`  - hire you